automatic/waterfox/update.ps1

```diff
@@ -25,23 +31,40 @@ function global:au_SearchReplace {
       "(?i)(^\s*softwareName\s*=\s*)'.*'" = "`${1}'$softwareName'"
       "(?i)(^\s*packageName\s*=\s*)'.*'"  = "`${1}'$($Latest.PackageName)'"
     }
+    ".\waterfox.nuspec"  = @{
+      "(?i)(^\s*\<id\>).*(\<\/id\>)" = "`${1}$($Latest.PackageName)`${2}"
```

Comment:
Please remove this regex. The package id will already be set to the `PackageName` property used in the stream.

Reply:
done